multiqc/modules/cellranger/count.py

```diff
@@ -90,12 +108,22 @@ def parse_count_html(self):
                 )
 
             self.add_section(
-                name="Count - Summary stats",
+                name="Gene Expression - Summary stats",
```

Comment:
Why renaming only here, but not the other sections prefixed with "Count"?

Reply:
I'd keep the old name for now, if you don't mind

Reply:
Makes sense, thanks!